lib/evmone/execution_state.hpp

```diff
@@ -181,5 +185,12 @@ struct ExecutionState
     }
 
     bool in_static_mode() const { return (msg->flags & EVMC_STATIC) != 0; }
```

Comment:
Sanitizers want [[nodiscard]] on this ontouched line.

Reply:
This is because we added public/private specifiers and clang-tidy do not consider it as a record type any more. Fixed.